src/main/java/uk/co/ramp/covid/simulation/Model.java

```diff
@@ -204,29 +223,44 @@ public boolean isValid() {
 
         return stats;
     }
-    
-    private void writeOutput(int iterId, List<List<DailyStats>> s) {
-        Path outP;
 
+    private void configureLoggerRedirects() {
```

Comment:
Ideally, I would put this method in another class.  (Perhaps in the output package.)